packages/stylelint-plugin-dialtone/lib/rules/use-dialtone-tokens.js

```diff
@@ -0,0 +1,63 @@
+const stylelint = require("stylelint");
+
+const {
+  createPlugin,
+  utils: { report, ruleMessages, validateOptions },
+} = stylelint;
+
+const ruleName = '@dialpad/stylelint-plugin-dialtone/use-dialtone-tokens';
+
+const messages = ruleMessages(ruleName, {
+  useDialtoneTokens: (value) => `Use a Dialtone token instead of "${value}".
+    A list of available tokens can be found here: https://dialtone.dialpad.com/tokens.`,
+  useRemInsteadOfPx: (value) => `If it's not possible to use a Dialtone token, use rem instead of px for "${value}".`,
```

Comment:
Can we suggest the rem value to replace it with?

ex
`If it's not possible to use a Dialtone token, use ${value/10.0}rem instead of ${value}px.`

Reply:
Let's say to consult with design on which token should be used, and only if the designer approves it should we use rem instead of a token. In the case of this decision, they can then set a lint ignore on this line.